ChatApp/src/screens/Profile/profile.tsx

```diff
@@ -0,0 +1,404 @@
+
+const Profile_Screen = ({ navigation }: any) => {
+  const [modalVisible, setModalVisible] = useState(false);
+  const [name, setName] = useState('');
+  const [nickName, setNickName] = useState('');
+  const [dateOfBirth, setDateOfBirth] = useState('');
+  const [email, setEmail] = useState('');
+  const [about, setAbout] = useState('');
+  const [uploading, setUploading] = useState(false);
+  const [transferred, setTransferred] = useState(0);
+  const [imageUrl, setImageUrl] = useState('');
+  const [date, setDate] = useState(new Date());
+  const [mode, setMode] = useState('date');
+  const [show, setShow] = useState(false);
+  const [fileUri, setFileUri] = useState('');
+
+  //select image from device
+
+  const chooseImage = () => {
+    let options = {
+      title: 'Select Image',
+      customButtons: [{ name: 'customOptionKey', title: 'Choose Photo from Custom Option' }],
+      storageOptions: {
+        skipBackup: true,
+        path: 'images',
+      },
+    };
+    ImagePicker.launchImageLibrary(options, (response: any) => {
+      console.log('Response = ', response);
+
+      if (response.didCancel) {
+        console.log('User cancelled image picker');
+      } else if (response.error) {
+        console.log('ImagePicker Error: ', response.error);
+      } else if (response.customButton) {
+        console.log('User tapped custom button: ', response.customButton);
+        Alert.alert(response.customButton);
+      } else {
+        const source = { uri: response.uri };
+
+        // You can also display the image using data:
+        // const source = { uri: 'data:image/jpeg;base64,' + response.data };
+        // alert(JSON.stringify(response));s
+        console.log('response====', JSON.stringify(response.assets[0].uri));
+        setFileUri(response.assets[0].uri);
+      }
+    });
+  };
+
+  //  capture image from camera
+
+  const launchCamera = () => {
+    let options = {
+      storageOptions: {
+        skipBackup: true,
+        path: 'images',
+      },
+    };
+    ImagePicker.launchCamera(options, (response: any) => {
+      console.log('Response = ', response);
+
+      if (response.didCancel) {
+        console.log('User cancelled image picker');
+      } else if (response.error) {
+        console.log('ImagePicker Error: ', response.error);
+      } else if (response.customButton) {
+        console.log('User tapped custom button: ', response.customButton);
+        Alert.alert(response.customButton);
+      } else {
+        const source = { uri: response.uri };
+        console.log('response', JSON.stringify(response));
+        setFileUri(response.uri);
+      }
+    });
+  };
+
+  // rendering image
+
+  const renderFileUri = () => {
+    if (imageUrl) {
+      return <Image source={{ uri: imageUrl }} style={Styles.images} />;
+    } else {
+      return <Image source={require('../../../assets/images/dummy.png')} style={Styles.images} />;
+    }
+  };
```

Comment:
This can be re-written as 
  const renderFileUri = () => <Image source={imageUrl ? { uri: imageUrl }: require('../../../assets/images/dummy.png')} style={Styles.images} />